### DISCUSSION TITLE
Change borg encryption password

### DISCUSSION THREAD

**Discussion by lum4chi** [nextcloud/all-in-one#5707] [Questions] [answered]
Hi, I would like to change the encryption password and start a new backup from an existing Nextcloud AIO instance. 
So far I tried to manually change `backupsecret` with a new key, remove `borg.config` in order to reset backup path and restart the mastercontainer but somehow old secret is still being loaded.

Any help is appreciated, thank you!

**Comment by szaimen** [comment, marked as answer]
Hi, the password is stored inside `/var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data/data/configuration.json` as `BORGBACKUP_PASSWORD`. You can change it in there.
